docs/concepts/architecture/apiserver-aggregation.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Extending the Kubernetes API with Aggregator
+assignees:
+- chenopis
+---
+
+{% capture overview %}
+
+The *Aggregator* is a system to allow the Kubernetes apiserver to be extended with additional APIs, which are not part of the core Kubernetes APIs. 
```

Comment:
I would like to refer to this as the "aggregation layer" since it is built in and not a separate component.